### PR TITLE
WebpackDevServerService

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Add WebpackDevServerService to wdio services. This service allows webpack-dev-server to start and return a promise when the webpack compiler is complete. 

With the addition of this service, 
- Update nightwatch config to use WebpackDevServerService
- Update tests structure to WebpackDevServerService
- Update wdio tests to be under 'wdio' dir instead of 'specs' dir

### CODE REVIEW

File: src/wdio/services/WebpackDevServerService.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import webpack from 'webpack';
+
+export default class WebDevServerService {
+  async onPrepare(config) {
+    if (!config.webpackConfig) {
+      return;
+    }
+
+    this.webpackConfig = config.webpackConfig;
+    this.port = config.webpackPort || 8080;
```
Comment (mhemesath) [inline src/wdio/services/WebpackDevServerService.js:13]: I'd expect this to be bundled in the webpackConfig object

File: src/wdio/services/WebpackDevServerService.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+      this.compiler = webpack(this.webpackConfig);
+
+      this.compiler.plugin('done', () => {
+        // eslint-disable-next-line no-console
+        console.log('[WebDevService] Webpack compiled successfully');
+        resolve();
+      });
+
+      this.compiler.plugin('invlaid', () => {
```
Comment (ryanthemanuel) [inline src/wdio/services/WebpackDevServerService.js]: invalid not invlaid

File: src/wdio/services/WebpackDevServerService.js (hunk 1 above)
Comment (mhemesath) [reply src/wdio/services/WebpackDevServerService.js:13]: actually, nevermind.. i get why you did it this way

**Comment by mhemesath** [comment]
Snapshot files shoudln't be checked, in, i thought i had an ignore on that

**Review by mhemesath — APPROVED** [review]

**Comment by mhemesath** [comment]
Nice work, this looks great

**Review by ryanthemanuel — APPROVED** [review]

**Review by mhemesath — APPROVED** [review]

**Pull request merged**